tests/test_fits.py

```diff
@@ -14,6 +14,25 @@
 my_max = 1000  # Typically number of galaxies in loop
 except_int = 750  # Where test exception occurs in loop
 filename = "MyFile.Fits"
+cls_file = "Cls.npz"
+
+
+def test_read_write_cls(rng: np.random.Generator, tmp_path: pathlib.Path) -> None:
+    cls = rng.normal(size=(10, 10))
+    save_cls(tmp_path / cls_file, cls)
+
+    assert pathlib.Path.exists(tmp_path / cls_file)
+
+    with np.load(tmp_path / cls_file) as npz:
+        values = npz["values"]
+        split = npz["split"]
+
+    np.testing.assert_allclose(values, np.concatenate(cls))
```

Comment:
Why `assert_allclose` here? Wouldn't https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html be more appropriate?

Reply:
Oh yes, thanks!